Activities/Shared/UiPath.Shared.Activities/EnumExtensions.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.CodeDom;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace UiPath.Shared.Activities
+{
+    internal static class EnumExtensions<T> where T : struct
+    {
+        public static bool EnumTryParseNoIntegers(string value, out T result)
+        {
+            if (int.TryParse(value, out _))
```

Comment:
this code was trying to eliminate integer values and allow only enum names, because Enum.TryParse will return true for non-existent integer values (it will return that value as if it's part of the enum)

anyway, the method was copied from system but it was useless so I've dropped it